CMakeLists.txt

```diff
@@ -103,9 +103,11 @@ elseif(BASEARCH_X86_FOUND)
     option(WITH_SSE4 "Build with SSE4" ON)
     option(WITH_PCLMULQDQ "Build with PCLMULQDQ" ON)
 endif()
+option(WITH_INFLATE_STRICT "Build with strict inflate distance checking" OFF)
+option(WITH_INFLATE_ALLOW_INVALID_DIST "Build with zero fill for inflate invalid distances" OFF)
```

Comment:
These should be advanced options (and moved up over the basearch bunch)

Reply:
It is already part of `mark_as_advanced`

Reply:
I have moved those two lines above basearch bunch as you suggested.